apps/modeler/src/main_window.cpp

```diff
@@ -282,7 +282,7 @@ pcl::modeler::MainWindow::updateRecentActions(
   }
 
   recent_items.removeDuplicates();
-  int recent_number = std::min(int(MAX_RECENT_NUMBER), recent_items.size());
+  int recent_number = std::min<int>(int(MAX_RECENT_NUMBER), recent_items.size());
```

Comment:
Why is it necessary to explicit tell std::min to use int?

Reply:
In Qt6, recent_items.size() returns a qsizetype, which is usually not equal to int (more like ssize_t). So min doesn't know which type to use.